samples/apps/copilot-chat-app/webapi/Controllers/SemanticKernelController.cs

```diff
@@ -119,4 +125,17 @@ public class SemanticKernelController : ControllerBase
 
         return this.Ok(new AskResult { Value = result.Result, Variables = result.Variables.Select(v => new KeyValuePair<string, string>(v.Key, v.Value)) });
     }
+    private async Task RegisterOpenApiSkillsAsync(OpenApiSkillsAuthHeaders openApiSkillsAuthHeaders, IKernel kernel)
```

Comment:
The planner registers skills for itself through a factory: https://github.com/microsoft/semantic-kernel/blob/main/samples/apps/copilot-chat-app/webapi/SemanticKernelExtensions.cs#L95

With your pattern here we may need to rearchitect how we create the planner. Let's chat tomorrow about it!

Reply:
RegisterPlannerSkillsAsync now calls RegisterOpenApiSkillsAsync